src/crab_utils/adapt_sgraph.hpp

```diff
@@ -8,8 +8,10 @@
 #include "crab_utils/debug.hpp"
 // Adaptive sparse-set based weighted graph implementation
 
+#ifdef __GNUC__
 #pragma GCC diagnostic push
 #pragma GCC diagnostic ignored "-Wsign-compare"
+#endif
```

Comment:
Seems like this may be simply removed. At least that's the case on my machine.

Reply:
Was trying to minimize changes in this PR.  Do you want it removed in this PR?

Reply:
If it can be removed, then yes. I can also remove it "silently" directly on master if you prefer it that way.

Reply:
Done